app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -210,7 +206,8 @@ def finish(*_args)
     # Dummy method, nothing to execute here. Job finished.
   end
 
-  alias_method :abort_job, :cleanup
+  alias_method :abort_job,    :cleanup
+  alias_method :initializing, :dispatch_start
```

Comment:
It isn't. The old one cause the job to transition straight from initializing to start.
The new one causes the job to be picked up by job_proxy_dispatcher by setting ```dispatch_status = "pending"```
there also ```@storage_dispatcher_process_finish_flag = false``` there and I have no idea what it does :open_mouth: